go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/vthiery/healthcheck
```

Comment:
This module should be called `github.com/etherlabsio/healthcheck`

Reply:
Indeed :+1: